includes/class-amp-theme-support.php

```diff
@@ -100,6 +100,8 @@ public static function init() {
 		 * action to template_redirect--the wp action--is used instead.
 		 */
 		add_action( 'wp', array( __CLASS__, 'finish_init' ), PHP_INT_MAX );
+
+		add_action( 'wp_enqueue_scripts', array( __CLASS__, 'enqueue_assets' ) );
```

Comment:
Putting this here caused a regression where the `amp-runtime` is enqueued even on non-AMP pages. PR to fix in https://github.com/Automattic/amp-wp/pull/1033